include/swift/SIL/SILBuilder.h

```diff
@@ -2058,53 +2058,56 @@ class SILBuilder {
   }
 };
 
-/// An wrapper on top of SILBuilder's constructor that automatically sets the
-/// current SILDebugScope based on the specified insertion point. This is useful
-/// for situations where a single SIL instruction is lowered into a sequence of
-/// SIL instructions.
-class SILBuilderWithScope : public SILBuilder {
+/// A wrapper on top of SILBuilder's constructor that sets the insertion point,
+/// and automatically sets the SILDebugScope based on the specified IP. This is
+/// useful for situations where a single SIL instruction is lowered into a
```

Comment:
I would be even more affirmative:

`Use this variant of SILBuilder when a single ...`